arbnode/transaction_streamer.go

```diff
@@ -232,17 +232,19 @@ func (s *TransactionStreamer) ReorgAt(firstMsgIdxReorged arbutil.MessageIndex) e
 }
 
 func (s *TransactionStreamer) ReorgAtAndEndBatch(batch ethdb.Batch, firstMsgIdxReorged arbutil.MessageIndex) error {
-	s.insertionMutex.Lock()
-	defer s.insertionMutex.Unlock()
-	err := s.addMessagesAndReorg(batch, firstMsgIdxReorged, nil)
-	if err != nil {
-		return err
-	}
-	err = batch.Write()
-	if err != nil {
-		return err
-	}
-	return nil
+    // Only acquire insertionMutex here. The reorgMutex is handled inside addMessagesAndReorg.
```

Comment:
Now, I'm confused again. This looks like it's only adding a comment and changing the indentation?
Did you run this through go fmt?